src/components/About/MissionVision.tsx

```diff
@@ -0,0 +1,23 @@
+const MissionContent = `Lorem ipsum dolor sit amet, consectetur adipiscing elit.`
+const VisionContent = `Lorem ipsum dolor sit amet, consectetur adipiscing elit.`
+
+export default function MissionVision() {
+  return (
+    <section className="flex px-[15rem] py-[7.5rem] gap-x-[4.56rem]">
+      {
+        [["Mission", MissionContent], ["Vision", VisionContent]]
+        .map(([title, content]) => (
+          <div key={title}>
+            <div className="flex items-start gap-x-[1.5rem] mb-[2.5rem]" key={title}>
```

Comment:
you could remove the key here I guess

Reply:
Regarding with the `key`, without it, it produces the following warning:
```
Warning: Each child in a list should have a unique "key" prop.

Check the top-level render call using <section>. See https://reactjs.org/link/warning-keys for more information.
    at div
    at MissionVision
    at AboutPage
    at main
    at div
    at Layout (webpack-internal:///./src/components/Layout.tsx:17:19)
    at App (webpack-internal:///./src/pages/_app.tsx:22:16)
    at StyleRegistry (/home/datsudo/Documents/dev/tpg-web/node_modules/styled-jsx/dist/index/index.js:449:36)
    at PathnameContextProviderAdapter (/home/datsudo/Documents/dev/tpg-web/node_modules/next/dist/shared/lib/router/adapters.js:60:11)
    at AppContainer (/home/datsudo/Documents/dev/tpg-web/node_modules/next/dist/server/render.js:290:29)
    at AppContainerWithIsomorphicFiberStructure (/home/datsudo/Documents/dev/tpg-web/node_modules/next/dist/server/render.js:326:57)
    at div
    at Body (/home/datsudo/Documents/dev/tpg-web/node_modules/next/dist/server/render.js:613:21)
```

Should I disregard this for now?

Reply:
can you push that branch from your forked repo here in this repo so that it's easy to navigate and try out?

Reply:
The key that you removed was from the outer div (line 10) and not the inner div (line 11) that's why there is that error. Key props should be in the parent div enclosing all the children inside it in a `map` function which in this case is the outer div (line 10)

Reply:
> The key that you removed was from the outer div (line 10) and not the inner div (line 11) that's why there is that error. Key props should be in the parent div enclosing all the children inside it in a `map` function which in this case is the outer div (line 10)

Oh sorry I didn't notice that lol

So the key in outer `div` will stay?

Reply:
> can you push that branch from your forked repo here in this repo so that it's easy to navigate and try out? Or I think I can check it out without the need to push it

Is this fine? I did my changes in my fork to make the main repo clean. I think there 

> can you push that branch from your forked repo here in this repo so that it's easy to navigate and try out? Or I think I can check it out without the need to push it

Done